lib/flutter_animation_progress_bar.dart

```diff
@@ -77,6 +77,7 @@ class _FAProgressBarState extends State<FAProgressBar>
           .animate(_controller);
     });
     _controller.reset();
+    _controller.duration = widget.animatedDuration;
```

Comment:
I think to duration has already set at line 53

Reply:
Yes. It is set when the progress bar is initialized.
Re-setting it here allows to change the duration later (if value is the same it does nothing).
In my use case I have to show progress when screen is opened without animation. Later user can interact with the screen and the value change is animated.

Reply:
Thanks for your clarifying, I'll merge to the main branch and publish new version soon

Reply:
It's awesome. Thank you 🎉